src/components/Lesson/LessonButton.js

```diff
@@ -0,0 +1,29 @@
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import styles from './Lesson.scss';
+import {getTranslator} from '../../selectors/translate';
+import {getReadmepathFromLessonpath} from '../../util';
+import InstructionButton from './InstructionButton';
+
+const LessonButton = ({buttonPath, t}) => {
+  const buttonText= t('lessons.tolesson');
+  const bsSize = 'small';
+  const className = styles.buttonMargin;
+  return <InstructionButton {...{className, buttonPath, buttonText, bsSize}}/>;
+};
+
+LessonButton.propTypes = {
+  // mapStateToProps
+  buttonPath: PropTypes.string,
+  t: PropTypes.func.isRequired
```

Comment:
Hm, bare et pirkeforslag, men kanskje i stedet for å sende inn `t`, kunne man sende inn `buttonText`? Så kunne du ha følgende i mapStateToProps:
```
buttonText: getTranslator(state)('lesson.tolesson'),
```

Reply:
Brukes dette noe annet sted? Ellers lar jeg det være som det er nå